### DISCUSSION TITLE
When you say "Link opengl if you get linking errors" what do you mean by that?

### DISCUSSION THREAD

**Discussion by KitChaos-DreamscapeStudios** [SFML/imgui-sfml#259] [Q&A] [answered]
![image](https://github.com/SFML/imgui-sfml/assets/97907903/227591fa-d807-46f4-9a6e-b989145d9712)
Followed the set up instructions for both imGUI and this, get this. I'm not sure what is meant when you say "Link opengl". Do you mean include it in the include directories?

**Comment by eliasdaler** [comment, marked as answer]
Add "opengl32.lib" to your Configuration Properties > Linker > Input > Additional Dependencies

**Reply by KitChaos-DreamscapeStudios** [reply]
thank you!
